lib/assessment.rb

```diff
@@ -13,28 +13,36 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(&:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    if numbers.each.odd?
+      true
+    else
+      false
+    end
```

Comment:
There is another method you should use here - one which determines if at least some elements in a given set pass some test. Can you think of what it might be?